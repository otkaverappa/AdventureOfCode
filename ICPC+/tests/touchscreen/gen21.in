19
ljvoxgzjbtjkimpvsxoqi 10
aswinzeumyhxvxwbzgfsp
pacexudlmmnekvtjcnxaw
uvusztyenqvpubnqhacto
gfjzolpnnljlhbkccqpsn
foouevugqiovuplheyvsj
gbnsepkvjxcxnwdltjbdz
zzokhtlenwmojguaqslpu
kctnnmyrrxqrmcyhnewkq
mvyhwozhdvuhqhxegqwfg
ntinaytykllfjudxtncqh
lhqilwyjegwxrlareybpj 10
iafmxabbrinayxnxyxbgv
axhaonbrfmafsofspgjxv
jxvxwvwxbrybyarobjtol
agzfyqokplvoiuldqihii
iilbwmmrcxukyblqpxfds
ulpfrfpypyialjyxwscvm
mwnzmfwskpoxgtolkodky
dvkbvgvxdhlzvlochgmyw
jerasboxmnchxdefbbkip
kfaahjivkefqvfkyvhkij
uhpamqcxytjgtjnerlqxq 10
hsxtssbfbmzibzxtdwuwh
arspyjaqisxcpruiuzlga
vkcsdhqxexzxprxaroikn
kbgfjagujhstjmyqdywax
wsotsijduxpvfufgbbrkv
ktikjliiniehyxcijfcgu
ylrftuikeewzojibtqobw
ycuakffpnbnyuuuqegukf
vyogzjyqlvoprobwvqjyf
jvbflgmistdtjjssikffa
uwqysnodmwmjxsufgeyzi 10
ritlcdyiiycepayerbsnq
bkiysqcqrmjzgublsluto
aiqbojehyuilegdujtnvc
mdyqrtbnmrpwkqkvvrura
fwiistdgrhuufmlyqymho
itgtpdkjvmqtvanodvfks
crepswqeegnzmiqqsbnfr
icrvswtyilcarruohaunp
tcxmusnhzfpbymtvftdsv
fmnzawcwmrropdihrpgwg
jwucrcyvutcgieihggtxy 10
hobpxugdsmopitgkrdfmg
nxkyeqexpeggivdcdjvrx
mzqskhyruegtqexilwxpc
fxxjcaszrplsgdcpbtlha
exgbhrzhjbmiavkcnjtfw
mlaqadkonmskswsmvzvxl
fxjsawdvdajoccrfmfsaz
fsvzgtadqljpwdrvgpyga
nicgqqnkqmplkordqvvcg
kajevpvuyvhiypaqclhoc
sytldjiynojoxpjpkdnhm 10
xfbyyflfwnzugmyqvyfjj
tjydykinsvkxykxevcajd
wrqwjlvpweiffnfpwshrf
hrrevnjxymuresbrpsouc
vzqbqmvyfcfyujujutjhp
alidfxxttaqusrlgplnrq
lnbhywbhfqksapxypqtpj
ohcbprmclenafvzdwgkos
dpkapztiqcyxezmynqjrd
jzaiewrpnlscvurunzkra
jwbxuppfitojtzossffdx 10
hbraxhzkybvwdvqukvcdm
mzncrfhykgflzfjggthko
dnjuhwrlzfzasbrzkpkqw
vqdgymbfwsjlbfszydzed
zygtyrkkjjfbmoaaphyhs
jlzbkyhjemlesgelqovzw
wokyozinhaysaacajoevb
knhpzzdwzzspltdlbsubr
odtsdcgjxjumsllsqhrsc
idvnqygkzzyfuriwzuvjq
hddvvteoliwrfkhdssfrs 10
knlullqgwgpzjmvftjqbi
jhsrmklrddbqrybcojava
zgnwlgiejqnskgfurwxxa
noyqsmzsjztsosfucjguz
yehfyzexyekocagqbacav
uppbltntnmsttztudquhc
ilcqedsigqcvfdibqcoew
kavdwburkwzybscgkkmco
jktslmubsslsnrrolibjj
zmbbtonhqdtawnnkjpdba
xorofewgqgfcjixzxhqaa 10
swngjwvmxxjnryuxwcnej
sorppqwfqwapliyhdkgcv
uvvqtstiwcamwscoojfnj
warwkwiqbdmyzdttycpce
eawgpnrwcdscwomvwfwbr
wawqtwslzwpzvxqiommte
oqudlqkjsdfuavqwpbxms
xjpprfcdzgtpdyctilnms
iooamdbkrtjakytrayqju
hmsjhdwxrpfhfhtkifcdo
enpygryxcsgqnsysqvjfc 10
qmjlxtrzwhflydrqeqswx
llrjdhgonkgbvsyolzmsf
xtkqjohdlgowxzcehqtrx
vmpvccvouannkgyzpemvu
kuwmaddwxatlrootjeplu
fwafvpjjmgtgcgihleenz
qeogzzkolevjebgvmphsi
nwqyedekqeawuivvtmizj
rgkxbzmjtxyqpwwtdglhj
idrfykthmdyspyvqmekjf
cbdywgghnprqhxrtsyfvy 10
znwvdjbpuirxmrtsybins
awrrqkrxhszwowazxrwgj
tucpmaquqkxoeqgoiexaf
uqbusanogwjabypdqlvdi
khzqvjwsjbobektfyjnwv
oxvfbmqwpbiyabtlxnwbe
xfotmmeajzqgvvjjniyoq
yqrrbrgxskxzbqonwowwe
ftbocowcfpavgtzxbwrov
tpnhckxbidgbgufuthbij
xocwofuhtqaifjnpioxnu 10
atqfqjmrrvoizmwfjfabi
lgrywcovpkwjabzkqqdlh
lnvjserufzfmsdkutfkdb
vefuqvmvgvjvtupxlkfkp
tdvdxoijsmgyrcqmpnumy
qftfferkrhducitsqfmel
mvoehfsetqwbjdgqvsjcx
fgfayvflatxxjchqwljmh
kymsqhkzjhepmeqkmbmfa
jqcqgydrnmelzwbgjassg
jfkzrwcfddovfhbgmvtbz 10
gavhigjczpmgblxaqefts
babegqzbrahryqbezedqq
jsdiutmzohapkhvaixrjg
khxmlwqrpicjnndjbcxkf
mvoixwhrfpbmmpalhtapv
lfkoomslyxytlitkrkpgl
evbggkziawthjjvvbgtbg
poqkyjuqshunkavvbdxzy
eihzfihblqsagcyswokfk
xqmsnpvkpvoxcpfmwgyoy
yvaynypxfbwxookdluugk 10
ukzzwyfwmfwjhvyhmxmnt
mdhwhurdcdxmdyldfhpke
ztbxapvpcqbgazpussxws
lbqweygtimuenugervghy
ohadesybosopknvitptfl
xufdywbgebvnbaruauksj
afwvnpmiwyfsflscmyinv
vrynlyhvtsxyovnfjwdjd
xpoptaqbqlyjjnukxrdpr
dgosnzotjuqaigtlywdjw
mvkifjbkaspgihwiyptsf 10
tcnonlmqxicsukzdojegy
mrhizfzuzhncudpipihxm
zuzaxnjdvjqmsanaahzhw
ddzvnoduortiqvfdgjzrb
nkbbkbklkhpogmdvpyjgt
rzpycvjdnktzmwjpgwcnn
swzvrqtcxnvyetczfgopb
ndxyvdvxtippikgygkxvf
vzayafgpxhcaebvjyucgj
rqwzpclozskytkyaspxzu
afxxqvsvdbouzojdbxcuj 10
aptbpnspmmptlmmgehmiv
ghlplnonjaqyvtnjncxct
rngfwmmiwjoevfrktgtty
tptgahlxjgqwovsbhcxtq
dovwarfvmdodxwdgjbpqr
nenihwkhpbkfyjfroaetq
hsnnywopmfdtularxhiau
ozdvstywmogfdtdajvoqy
hllkeirojlckqacjybwoj
dsciundkdduqoeuzxlklp
ubpynpblekoyoysbceffy 10
xweuvcffnubokbdzdofnc
debxffcmmckihefjjmwdn
mprqrugwjjanmzvtbhfer
onxvwgiumvidmauihrust
hesezwmecfupfqnnyjzvr
fkszubqotlyzgcycqegyz
vdrkqptsmkxyewthpjacj
bimzkeeqcfnfwxynsqbdn
ajlvraetfovncwagaslhg
qdfoqygsdtuofphhvboly
cnucwxuheczikqaiwsnqo 10
bxdjgbmvmkzcedadxjici
smyswvnjnbmmhxuijrwvq
ycvafvjnxshlsahppqcse
qzcnjlegjxglshqnqgmin
xbpesgwxzdnyfckshqbgw
maffqwncgccjsicafzzko
aprkhaaliyxigeacrfiuh
sopwowvqgjqvcafccqmcp
viuxkncvjjpzblqxiggtw
dwyiacaneqkpmjbzmwlxn
kyaayiiecghaescgsrnkd 10
cwnfyzejwrvvswtcgxeng
gtyjatcpdgtatyyscjowe
locgsjdzymfrmqrftijzb
jscjmgsccyosbulmakkor
baivgddpehayjlkrfmudc
mgxzszjendgnndvqimwpm
uazesesokvaqucltnriqz
vdcrvldrcslftrzyknkhp
cbrnwfhexgbbluwyaoauz
hnrgldvxklanenjlwpjdq
