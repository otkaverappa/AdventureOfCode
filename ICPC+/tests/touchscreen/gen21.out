pacexudlmmnekvtjcnxaw 72
gfjzolpnnljlhbkccqpsn 75
mvyhwozhdvuhqhxegqwfg 75
ntinaytykllfjudxtncqh 78
zzokhtlenwmojguaqslpu 78
aswinzeumyhxvxwbzgfsp 79
foouevugqiovuplheyvsj 79
uvusztyenqvpubnqhacto 79
kctnnmyrrxqrmcyhnewkq 96
gbnsepkvjxcxnwdltjbdz 111
kfaahjivkefqvfkyvhkij 68
agzfyqokplvoiuldqihii 76
iafmxabbrinayxnxyxbgv 82
dvkbvgvxdhlzvlochgmyw 85
jerasboxmnchxdefbbkip 85
jxvxwvwxbrybyarobjtol 85
axhaonbrfmafsofspgjxv 87
ulpfrfpypyialjyxwscvm 87
mwnzmfwskpoxgtolkodky 97
iilbwmmrcxukyblqpxfds 105
kbgfjagujhstjmyqdywax 55
ycuakffpnbnyuuuqegukf 64
jvbflgmistdtjjssikffa 67
wsotsijduxpvfufgbbrkv 79
vyogzjyqlvoprobwvqjyf 82
ktikjliiniehyxcijfcgu 87
ylrftuikeewzojibtqobw 87
vkcsdhqxexzxprxaroikn 89
hsxtssbfbmzibzxtdwuwh 91
arspyjaqisxcpruiuzlga 103
fwiistdgrhuufmlyqymho 71
icrvswtyilcarruohaunp 71
mdyqrtbnmrpwkqkvvrura 72
tcxmusnhzfpbymtvftdsv 77
aiqbojehyuilegdujtnvc 79
fmnzawcwmrropdihrpgwg 84
itgtpdkjvmqtvanodvfks 85
ritlcdyiiycepayerbsnq 85
crepswqeegnzmiqqsbnfr 86
bkiysqcqrmjzgublsluto 93
nxkyeqexpeggivdcdjvrx 59
exgbhrzhjbmiavkcnjtfw 66
fxjsawdvdajoccrfmfsaz 71
mzqskhyruegtqexilwxpc 75
fsvzgtadqljpwdrvgpyga 78
mlaqadkonmskswsmvzvxl 78
hobpxugdsmopitgkrdfmg 80
nicgqqnkqmplkordqvvcg 81
fxxjcaszrplsgdcpbtlha 84
kajevpvuyvhiypaqclhoc 84
hrrevnjxymuresbrpsouc 74
vzqbqmvyfcfyujujutjhp 74
xfbyyflfwnzugmyqvyfjj 76
lnbhywbhfqksapxypqtpj 77
jzaiewrpnlscvurunzkra 79
wrqwjlvpweiffnfpwshrf 79
tjydykinsvkxykxevcajd 82
alidfxxttaqusrlgplnrq 86
dpkapztiqcyxezmynqjrd 98
ohcbprmclenafvzdwgkos 107
odtsdcgjxjumsllsqhrsc 66
vqdgymbfwsjlbfszydzed 69
mzncrfhykgflzfjggthko 71
idvnqygkzzyfuriwzuvjq 73
jlzbkyhjemlesgelqovzw 80
zygtyrkkjjfbmoaaphyhs 87
hbraxhzkybvwdvqukvcdm 89
wokyozinhaysaacajoevb 90
dnjuhwrlzfzasbrzkpkqw 105
knhpzzdwzzspltdlbsubr 118
ilcqedsigqcvfdibqcoew 63
jhsrmklrddbqrybcojava 72
uppbltntnmsttztudquhc 72
yehfyzexyekocagqbacav 73
kavdwburkwzybscgkkmco 76
zgnwlgiejqnskgfurwxxa 78
zmbbtonhqdtawnnkjpdba 84
noyqsmzsjztsosfucjguz 85
knlullqgwgpzjmvftjqbi 92
jktslmubsslsnrrolibjj 100
sorppqwfqwapliyhdkgcv 58
xjpprfcdzgtpdyctilnms 70
eawgpnrwcdscwomvwfwbr 74
iooamdbkrtjakytrayqju 74
swngjwvmxxjnryuxwcnej 77
hmsjhdwxrpfhfhtkifcdo 79
oqudlqkjsdfuavqwpbxms 94
wawqtwslzwpzvxqiommte 95
uvvqtstiwcamwscoojfnj 96
warwkwiqbdmyzdttycpce 100
qmjlxtrzwhflydrqeqswx 59
rgkxbzmjtxyqpwwtdglhj 63
xtkqjohdlgowxzcehqtrx 69
vmpvccvouannkgyzpemvu 71
idrfykthmdyspyvqmekjf 74
llrjdhgonkgbvsyolzmsf 77
nwqyedekqeawuivvtmizj 81
fwafvpjjmgtgcgihleenz 84
kuwmaddwxatlrootjeplu 88
qeogzzkolevjebgvmphsi 90
znwvdjbpuirxmrtsybins 58
awrrqkrxhszwowazxrwgj 65
yqrrbrgxskxzbqonwowwe 68
uqbusanogwjabypdqlvdi 70
ftbocowcfpavgtzxbwrov 74
tpnhckxbidgbgufuthbij 74
oxvfbmqwpbiyabtlxnwbe 80
khzqvjwsjbobektfyjnwv 82
xfotmmeajzqgvvjjniyoq 84
tucpmaquqkxoeqgoiexaf 86
atqfqjmrrvoizmwfjfabi 74
fgfayvflatxxjchqwljmh 76
vefuqvmvgvjvtupxlkfkp 79
lnvjserufzfmsdkutfkdb 81
qftfferkrhducitsqfmel 81
jqcqgydrnmelzwbgjassg 84
tdvdxoijsmgyrcqmpnumy 84
lgrywcovpkwjabzkqqdlh 88
kymsqhkzjhepmeqkmbmfa 90
mvoehfsetqwbjdgqvsjcx 90
evbggkziawthjjvvbgtbg 58
babegqzbrahryqbezedqq 63
gavhigjczpmgblxaqefts 77
khxmlwqrpicjnndjbcxkf 78
mvoixwhrfpbmmpalhtapv 78
xqmsnpvkpvoxcpfmwgyoy 78
jsdiutmzohapkhvaixrjg 80
lfkoomslyxytlitkrkpgl 81
poqkyjuqshunkavvbdxzy 84
eihzfihblqsagcyswokfk 85
afwvnpmiwyfsflscmyinv 66
mdhwhurdcdxmdyldfhpke 66
vrynlyhvtsxyovnfjwdjd 69
lbqweygtimuenugervghy 70
ukzzwyfwmfwjhvyhmxmnt 76
dgosnzotjuqaigtlywdjw 82
ohadesybosopknvitptfl 84
xufdywbgebvnbaruauksj 84
xpoptaqbqlyjjnukxrdpr 109
ztbxapvpcqbgazpussxws 123
mrhizfzuzhncudpipihxm 56
ndxyvdvxtippikgygkxvf 64
vzayafgpxhcaebvjyucgj 70
tcnonlmqxicsukzdojegy 72
nkbbkbklkhpogmdvpyjgt 73
rqwzpclozskytkyaspxzu 84
rzpycvjdnktzmwjpgwcnn 87
ddzvnoduortiqvfdgjzrb 89
swzvrqtcxnvyetczfgopb 100
zuzaxnjdvjqmsanaahzhw 111
dovwarfvmdodxwdgjbpqr 69
rngfwmmiwjoevfrktgtty 71
aptbpnspmmptlmmgehmiv 78
dsciundkdduqoeuzxlklp 82
ozdvstywmogfdtdajvoqy 85
nenihwkhpbkfyjfroaetq 87
ghlplnonjaqyvtnjncxct 88
tptgahlxjgqwovsbhcxtq 90
hllkeirojlckqacjybwoj 93
hsnnywopmfdtularxhiau 96
mprqrugwjjanmzvtbhfer 79
xweuvcffnubokbdzdofnc 79
ajlvraetfovncwagaslhg 80
qdfoqygsdtuofphhvboly 80
debxffcmmckihefjjmwdn 82
vdrkqptsmkxyewthpjacj 85
bimzkeeqcfnfwxynsqbdn 86
fkszubqotlyzgcycqegyz 86
hesezwmecfupfqnnyjzvr 86
onxvwgiumvidmauihrust 86
smyswvnjnbmmhxuijrwvq 68
ycvafvjnxshlsahppqcse 71
maffqwncgccjsicafzzko 75
aprkhaaliyxigeacrfiuh 76
xbpesgwxzdnyfckshqbgw 76
sopwowvqgjqvcafccqmcp 77
bxdjgbmvmkzcedadxjici 78
dwyiacaneqkpmjbzmwlxn 83
qzcnjlegjxglshqnqgmin 84
viuxkncvjjpzblqxiggtw 100
vdcrvldrcslftrzyknkhp 70
jscjmgsccyosbulmakkor 71
mgxzszjendgnndvqimwpm 73
cbrnwfhexgbbluwyaoauz 74
locgsjdzymfrmqrftijzb 75
gtyjatcpdgtatyyscjowe 76
cwnfyzejwrvvswtcgxeng 77
hnrgldvxklanenjlwpjdq 83
uazesesokvaqucltnriqz 83
baivgddpehayjlkrfmudc 93
